src/Nethermind/Nethermind.Merge.Plugin/MergeConfig.cs

```diff
@@ -22,7 +22,7 @@ namespace Nethermind.Merge.Plugin
 {
     public class MergeConfig : IMergeConfig
     {
-        public bool Enabled { get; set; }
+        public bool Enabled { get; set; } = true;
```

Comment:
Maybe leave it at false then?

So either you force this flag or have TDD set (the latter is true for already merged networks). Not sure? Because now you have to force the flag false to disable it.

Reply:
If I change it to false we will still need to set this flag. We expect that for a not merged network it will work ( even if by default it is set to true). As long as you don't reach TTD, the node behaviour shouldn't change. However, I plan more testing around this.